app/views/restaurants/show.html.erb

```diff
@@ -45,10 +45,12 @@
 <%= link_to 'Back', restaurants_path %>
 
 <br />
-<% if current_user.try(:admin?) || current_user.id == @restaurant.user_id %>
-  <%= link_to 'Edit restaurant', edit_restaurant_path(@restaurant) %>
-  <br />
-  <%= link_to 'Delete restaurant', restaurant_path(@restaurant), method: :delete, data: { confirm: 'Are you sure?' } %>
+<% if current_user != nil %>
+  <% if current_user.try(:admin?) || current_user.id == @restaurant.user_id %>
```

Comment:
Great suggestion - we'll probably implement that in a new branch though, since this change was a temporary workaround and it doesn't really have to do with voting